store/generated_key/eigenda/eigenda.go

```diff
@@ -67,6 +67,22 @@ func (e Store) Get(ctx context.Context, key []byte) ([]byte, error) {
 	return decodedBlob, nil
 }
 
+// ToDo still not correct impl
+func (e Store) GetRaw(ctx context.Context, key []byte) ([]byte, error) {
```

Comment:
I'm confused, what is this supposed to do? It's doing the exact same thing as `Get()` right now.